tests/test_emails_helpers.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+import pytest
+
+from django.contrib.sites.models import Site
+
+from saleor.order.emails_helpers import (
+    get_order_confirmation_markup, get_organization, get_product_data)
+
+
+def test_get_organization(site_settings):
```

Comment:
Is it necessary to use this fixture if `Site.objects.get_current()` is called and modified below anyway? I think we can remove fixture OR use `site_settings.site.name = example_name`.
It is also to consider using Mock here, although mocking `name` can be tricky.

Reply:
I'd rather not mock anything unless completely necessary. Removed unused fixture

Reply:
Actually, after reading rest of the tests, I think we should use the fixture and use assignment `site = site_settings.site`, similar as we do in other tests - that's why we have this fixture. Calling model unrelated to the test that is actually referring to the `seetings.py` file, see:
https://github.com/django/django/blob/master/django/contrib/sites/models.py#L50
Our fixtures already uses pytest's `settings` fixture, so we should be safe here.
Moreover, call to `Sites.objects.get_current` is being used in more than one test.